dark/assets/css/styles.css

```diff
@@ -1051,6 +1051,12 @@ html {
   z-index: -1;
 }
 
+#contact .contact-text {

```

Comment:
Change the class from html of this @Rishav1707

Reply:
It is already there... @YoshithaRathnayake 

![Contact](https://user-images.githubusercontent.com/97666287/208665319-0d725235-8b7a-4871-b4f6-e59b94d57810.png)

Reply:
Ok then